.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Can you remove this, please? I'd prefer to avoid polluting the repository with tool-specific configuration files, unless strictly needed or really useful, but doesn't seem to be the case here. Thanks!

Reply:
yes sure